### DISCUSSION TITLE
5주차 스터디 회고

### DISCUSSION THREAD

**Discussion by alanhakhyeonsong** [Next-Level-Study/effective-java-study#15] [스터디 회고]
한 줄 회고와 건의사항을 말씀해주세요:)

**Comment by alanhakhyeonsong** [comment]
무작정 진도빼기보단 중간점검 차 공부했던 범위를 다시 리뷰하는 식으로 진행했는데 효과가 좋은 것 같습니다.
지난 번에 봤을 땐 넘어갔던 부분이 새롭게 보이는 것 같습니다.

**Comment by JeongJaew0n** [comment]
다시 보니까 이해가 더 잘됐습니다. @alanhakhyeonsong 님이 코드와 같이 설명해주니까 훨씬 이해가 잘 됐습니다. 역시 개발자는 코드로 말해야하나봐요

**Reply by alanhakhyeonsong** [reply]
검사합니다^^

**Comment by narangd0** [comment]
지금까지 배운 내용들을 통틀어 복습하고 재정리할 수 있어 좋았습니다.
또 앞으로 어떻게 공부를 하면 좋은지도 배워가는 것 같습니다.
다음주 제네릭부터 다시 파이팅합시다!

**Reply by narangd0** [reply]
복습하는 시간을 갖자고 제안하신 분의 통찰력이 참 뛰어난 것 같습니다!! ^ㅇ^ 굿

**Comment by sdh46** [comment]
리뷰를 하니 다시 보면서 이전 내용과 연관이 있었는데 미쳐 못챙겼던 부분이 보였고. 지난 시간에 미흡했던 부분들을 보충할 수 있는 기회가 되었습니다.

**Comment by Kurmarkpzgr** [comment]
평소 쓰는 자바는 구버전이고, 인텔리제이 인증 기한이 끝나서 요즘 실제로 코드로 실습해보는것에 어려움이 있었는데, 이렇게 보니 너무 좋았습니다.

**Comment by opgumo3** [comment]
복습을 하니, 처음 봤을 때 이해가 안갔던 부분이 이해가 가는 아주 좋은 결과가 있었습니다. 역시 복습!
적절한 분량 분배와 일정도 한 몫 하는 것 같습니다.

**Comment by SuJeong68** [comment]
복습해보며 이런 내용이 있었지 라는 생각을 했습니다. 코드로 다시 볼 수 있어서 좋았습니다.
